src/App.tsx

```diff
@@ -1,11 +1,13 @@
 import "./App.css";
 import About from "./components/organisms/About/About";
+import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
 import structure from "./structure.json";
```

Comment:
This should not be a direct import. Use js fetch API

Reply:
Data fetch added